src/ci/github-actions/ci.yml

```diff
@@ -633,7 +636,9 @@ jobs:
 
           - name: i686-mingw-2
             env:
-              RUST_CONFIGURE_ARGS: --build=i686-pc-windows-gnu
+              RUST_CONFIGURE_ARGS: >-
+                --build=i686-pc-windows-gnu
+                --set llvm.link-jobs=$MINGW_LLVM_LINK_JOBS
```

Comment:
please set this in one place around https://github.com/rust-lang/rust/blob/5e90940a4b742e7f1e86f21c26b56e99a8733458/src/ci/run.sh#L59-L60 rather than duplicating it across all jobs.

Reply:
Thanks for the pointer, I'll look into it 🙂